translation.rst

```diff
@@ -827,6 +827,69 @@ checks translation resources for several locales:
     add the missing translation to the log file. For details,
     see :ref:`reference-framework-translator-logging`.
 
+Switch Locale Programmatically
+------------------------------
+
+.. versionadded:: 6.1
+
+    The ``LocaleSwitcher`` was introduced in Symfony 6.1.
+
+Sometimes you need to change the locale of the application dynamically
+just to run some code. Imagine a console command that renders Twig templates
+of emails in different languages. You need to change the locale only to
+render those templates.
+
+The ``LocaleSwitcher`` class allows you to change at once the locale
+of:
+
+* All the services that are tagged with ``kernel.locale_aware``;
+* ``\Locale::setDefault()``;
+* If a request is available, the ``_locale`` request attribute.
```

Comment:
this is actually wrong. It updates the RequestContext of the Router (so that route generation autofills the `_locale` parameter, making i18n routes work for instance). But the code does not actually updates the Request object.

Reply:
Thanks, fixed in https://github.com/symfony/symfony-docs/pull/17774.